lib/flatpak-remote.c

```diff
@@ -160,6 +182,19 @@ flatpak_remote_class_init (FlatpakRemoteClass *klass)
                                                         "The name of the remote",
                                                         NULL,
                                                         G_PARAM_READWRITE));
+
+#ifdef FLATPAK_ENABLE_P2P
+#ifndef __GI_SCANNER__
+  g_object_class_install_property (object_class,
+                                   PROP_TYPE,
+                                   g_param_spec_enum ("type",
```

Comment:
Urgh, this is fallout from me marking the `FlatpakRemoteType` enum members as to be skipped, to avoid them getting picked up by `glib-mkenums` even when P2P support is disabled (because it ignores the `#ifdef FLATPAK_ENABLE_P2P` around them). I think the fix here is to unconditionally skip them, and manually add a `get_type()` function for them (conditionally).